GenderPayGap.UnitTests/GenderPayGap.WebUI.Tests/Controllers/Administration/AdminOrganisationScopeControllerTests.cs

```diff
@@ -0,0 +1,18 @@
+﻿using NUnit.Framework;
+
+namespace GenderPayGap.WebUI.Tests.Controllers.Administration
+{
+    [TestFixture]
+    [SetCulture("en-GB")]
+    public class AdminOrganisationScopeControllerTests
+    {
+
+        [Test]
+        [Description("POST: Old scopes are retired when a new scope is added")]
+        public void POST_Old_Scopes_Are_Retired_When_New_Scope_Added()
+        {
+            
+        }
+
+    }
+}

```

Comment:
Let's leave this file out unless/until it has some real tests in it

Reply:
Damn! This has been hanging around in my unstaged files for the other branch where I _actually_ plan to add some tests - didn't mean to commit it!